### ISSUE TITLE
prop-name-casing does not rename references in template

### ISSUE THREAD

**Issue by Ayyub-Omer** [vuejs/eslint-plugin-vue#863]
**Tell us about your environment**	
* **ESLint version:**        5.8.0
* **eslint-plugin-vue version:**	5.0.0
* **Node version:**    	11.9.0

**Please show your full configuration:**

```json
module.exports = {
  root: true,

  plugins: ["cypress"],

  env: {
    node: true,
    jest: true,
    "cypress/globals": true
  },

  rules: {
    "no-console": "off",
    "no-debugger": "off"
  },

  parserOptions: {
    parser: "babel-eslint"
  },

  globals: { grecaptcha: true },

  extends: ["plugin:vue/recommended", "@vue/prettier"]
};

```

**What did you do?**	

```vue
<template>
  {{ FooBar }}
</template>
<script>
/* eslint vue/prop-name-casing: [error, camelCase] */
export default {
  props: {
    FooBar: {}
  }
}
</script>
```
I ran npm run lint.

**What did you expect to happen?**
FooBar should be changed to fooBar in both the template and the script tags, or should not be changed at all.

**What actually happened?**

FooBar in props was renamed to fooBar, but FooBar in the template was left as is, and this broke my program because FooBar is not defined anymore.

```vue
<template>
  {{ FooBar }}
</template>
<script>
/* eslint vue/prop-name-casing: [error, camelCase] */
export default {
  props: {
    fooBar: {}
  }
}
</script>
```
